pkgs/development/python-modules/vprof/default.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, buildNpmPackage
+, buildPythonPackage
+, setuptools
+, wheel
+, psutil
+, pytestCheckHook
+, fetchFromGitHub
+}:
+
+let
+  pname = "vprof";
+  version = "0.38";
+
+  src = fetchFromGitHub {
+    owner = "nvdv";
+    repo = "vprof";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-K5N54MtnGi2ecyuDa/LiAYHQN+Hgkn6Jx3rExEQGnCU=";
+  };
+
+  ui-package = buildNpmPackage {
+    pname = "${pname}-ui";
+    inherit version src;
+
+    patches = [
+      # buildNpmPackage would complain about the package not having a version.
+      # So we add one manually
+      ./add-version-to-ui-npm-package.patch
+    ];
+
+    postPatch = ''
+      # See comment on the patch above
+      substituteInPlace package.json \
+        --replace "@version@" "${version}"
```

Comment:
Actually, I could do a simpler thing: hardcode the version to be `1.0` in the patch.
No need to add this useless substitution afterwards.
The version of the ui dependency has absolutely no impact.